ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.less

```diff
@@ -67,13 +71,19 @@
         vertical-align: top;
       }
     }
-
+    .is-split-button button,
```

Comment:
why wasn't this put in the split button component?

Reply:
It's already in the CSS, I didn't want to duplicate code and add yet another place to maintain the same functionality.